about.html

```diff
@@ -0,0 +1,146 @@
+          <li><a href="#Contact"> News</a></li>
+        </ul>
+      </nav>
+      <button type="button" id="header-button">Collision Campaign</button>
+    </header>
+    <div class="popup-menu">
+      <div class="menu-button">
+        <span class="hamburger"></span>
+        <span class="hamburger"></span>
+        <span class="hamburger"></span>
+      </div>
+      <ul>
+        <li>
+          <a href="index.html">Home</a>
+        </li>
+        <li><a href="about.html">About</a>
+        </li>
+        <li><a href="#Contact">News</a>
+        </li>
+      </ul>
+    </div>
+    <main class="main-content">
+      <section id="about-main">
+        <div>
+          <h1>"Hello! Tech World"</h1>
+          <h2>WEB Summit 2023</h2>
+          <hr class="solid gone" />
+          <p id="introduction">
+            Web Summit events have gathered half a million
+            people across Web Summit in Lisbon, Collision in
+            North America, and RISE in Asia since its
+            beginnings as a 150-person conference in Dublin
+            in 2009.
+            <span id="more-introduction"
+              >Web Summit’s mission has been to create
+              software that enables meaningful connections
+              between the CEOs, founders, investors, media,
+              politicians and cultural figureheads who are
+              reshaping the world.</span
+            >
+          </p>
+          <h3>
+            Please contact us per email for any further questions about the WEB SUMMIT 2023!
+          </h3>
+          <p id="email">info@torontosummit.com</p>
+        </div>
+      </section>
+      <section id="logo-reveal">
+        <h2>WEB SUMMIT 2023</h2>
+        <hr class="solid" />
+        <p>
+          The Global Summit brings together the community of experts and academics,
+          who compromise Collisions affiliate network in a different country every 
+          two years. This year, we hope to extend our invitation list including 
+          organisation, individuals who want to work with us on shared projects that
+          advance the cause of the Commons,free culture open knowlegde.
+          <span id="more-logo"
+            >So if you are active and engaged in the worlds of open content and knowlegde
+             - free software advocates, Wikipedians, Open knowlegde, galleries, libraries,
+             governments and foundations - we hope you will consider joining us this year
+             to build a stronger, more vibrant commons together</span
+          >
+        </p>
+        <div>
+          <img id="logo-image" src="images/Logo/logo.avif" alt="Logo" />
+        </div>
+      </section>
+      <section id="past-summits">
+        <h2>See the past WEB Summits</h2>
+        <hr class="solid" />
+        <p>
+          Take a look at the last two WEB Summits whick took place
+          in Lisbon and Dublin
+        </p>
+        <div id="past-pair">
```

Comment:
Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts).

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/164414698-30ba89da-4d63-49ac-a3a4-8747c42791c7.png) | ![image](https://user-images.githubusercontent.com/10332499/164414793-7798e522-50d9-4539-a8ae-90bf3505edb5.png)


Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/164415082-21027a5e-4417-4cd5-8710-a178e651f72f.png) | ![image](https://user-images.githubusercontent.com/10332499/164415329-30506a1e-d34f-4d94-8474-b185d8b8cc7d.png)


Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/164415582-8e188e45-3db3-4049-bca2-c8af5922f659.png) | ![image](https://user-images.githubusercontent.com/10332499/164415696-0b92c996-e4c9-4683-b724-ff046435e152.png)

Reply:
resolved